### PR TITLE
Add changelog for 3.56.0 release

### PR DESCRIPTION
**Pull request by melinath**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 ## 3.56.0 (Unreleased)
 
+NOTES:
+* billingBudget: doc updated for `google_billing_budget` ([#8316](https://github.com/hashicorp/terraform-provider-google/pull/8316))
```
Comment (c2thorn) [inline CHANGELOG.md:4]: I don't think this is worth adding to the changelog, unless it was going to call out something important that was added to the documentation. My bad for not catching it
```suggestion
```

**Review by c2thorn — CHANGES_REQUESTED** [review]
I think at least one PR that shows up in the google-beta logs is missing here:
https://github.com/hashicorp/terraform-provider-google/pull/8377

**Comment by c2thorn** [comment]
Or rather: it should be taken out of the google-beta release notes? I'll address it in the other PR

**Review by c2thorn — APPROVED** [review]

**Comment by melinath** [comment]
yeah I accidentally generated the beta changelog on master at first & missed that one in the cleanup - thanks for catching!

**Comment by melinath** [comment]
Fixed in the other PR

**Pull request merged**